### DISCUSSION TITLE
How to fix lonely question mark

### DISCUSSION THREAD

**Discussion by TwinkleToess** [LazyVim/LazyVim#3121] [Q&A]
Hi there! I have a lonely question mark there... How to fix that?  I'm using iTerm. The reddit community couldn't help me.
Im using agave nerd font and bunch of other. 

<img width="944" alt="image" src="https://github.com/LazyVim/LazyVim/assets/130213490/5d975d41-e63f-478c-bc9e-a840d5c8a526">

My ITerm config:

<img width="944" alt="image" src="https://github.com/LazyVim/LazyVim/assets/130213490/4e341ec4-2234-44c8-aa0b-c37972b2c6a8">

**Comment by dpetka2001** [comment]
See this [comment](https://github.com/LazyVim/LazyVim/issues/1139#issuecomment-1806858433) and apply the same thing to your configuration.
